copy/copy.go

```diff
@@ -358,6 +358,7 @@ func Image(ctx context.Context, policyContext *signature.PolicyContext, destRef,
 		return nil, fmt.Errorf("committing the finished image: %w", err)
 	}
 
+	logrus.Debugf("Copied manifest(s): %q", string(copiedManifest))
```

Comment:
I think this would be better handled by logging before the `PutManifest` call in `copyMultipleImages`; that should record the same data, and avoid logging the same manifest twice in single-architecture copy situations.